layouts/shortcodes/docs/instrumentation/span-status-preamble.md

```diff
@@ -0,0 +1,7 @@
+A [Status](/docs/concepts/signals/traces/#span-status) can be set on a
+[Span](/docs/concepts/signals/traces/#spans), typically used to specify that a
+Span has not completed successfully - `ERROR`. In rare scenarios, you could
```

Comment:
Is there any way for a back end to tell whether an OK status has been set manually on a span that contains errors? Otherwise, I'm not sure this practice should be even mentioned. Or if it's mentioned, it might be worth explaining what could be accomplished by doing this, or recommending to add additional metadata/attributes to a span so as to identify it as a former error.

Reply:
There isn't, no. Although it's definitely a valid use case -- for example, for a while there were some instrumentations that produces server spans which had a an error status when the status_code was 4xx. That's since been changed (also due to a spec change I think?) but in the interim there were definitely people overriding the status.